routers/api/v1/users.go

```diff
@@ -71,7 +73,7 @@ func (r *apiV1Router) Login(ctx *gin.Context) {
 
 	token, err := auth.NewJWT(*user, time.Now().Unix(), []byte(r.env.Get(environment.JWTSecret)))
 	if err != nil {
-		r.logger.Error("could not create JWT", zap.Error(err))
+		r.logger.Error("could not create JWT", zap.Any("user", *user), zap.Error(err))
```

Comment:
Does this log the whole user including the password?

Reply:
Oh yeah, nice catch!